transforms/transforms.go

```diff
@@ -262,3 +262,49 @@ func GroupBy(identifier RowIdentifier) optimus.TransformFunc {
 		return nil
 	}
 }
+
+// RowFilter is meant to return `true` if a section is meant to be filtered out.
+type RowFilter func(optimus.Row) bool
+
+// passthroughTable is a super simple optimus table that just passes rows through
+type passthroughTable struct {
+	c chan optimus.Row
+}
+
+func (p passthroughTable) Rows() <-chan optimus.Row {
+	return p.c
+}
+func (p passthroughTable) Err() error {
+	return nil
+}
+func (p passthroughTable) Stop() {}
+
+// BypassTransforms effectively wraps a slice of transform funcs with a gate to conditionally
+// apply the transforms only if they match the filter.
+func BypassTransforms(doBypass RowFilter, optionalTransforms []optimus.TransformFunc) optimus.TransformFunc {
+	outboundRows := make(chan optimus.Row)
+
+	// setup an optimus pipeline to process rows through all the provided optional transforms
+	passthroughChan := make(chan optimus.Row)
```

Comment:
It is messy to apply a `TransformFunc` due to their obtuse API: `func(in <-chan Row, out chan<- Row) error`. It is not meant to return until the input channel has been closed. [This is messy to do correctly](https://github.com/Clever/optimus/blob/v3.7.0/optimus.go#L70-L136).

Creating the table that has everything passed through seemed like a cleaner solution to avoid reimplementing the [`start`](https://github.com/Clever/optimus/blob/9caeeff3bb4965f14d6392fbed2a751a0d81c225/optimus.go#L70-L136) function.